public/app/features/transformers/calculateHeatmap/heatmap.ts

```diff
@@ -199,7 +199,7 @@ export function rowsToCellsHeatmap(opts: RowsHeatmapOptions): DataFrame {
         name: opts.value?.length ? opts.value : 'Value',
         type: FieldType.number,
         values: new ArrayVector(counts2),
-        config: yFields[0].config,
+        config: {}, // displayNameFromDS: undefined
```

Comment:
It appears that this is what is causing the test to break (snapshot) - not sure this is intended / desired (see [broken test here](https://github.com/grafana/grafana/blob/2ee748ec4c75f45601d6a93d1e96761a10a38084/public/app/features/transformers/calculateHeatmap/heatmap.test.ts#L51))